test/fork/merkle-streamer/MerkleStreamerLL.t.sol

```diff
@@ -133,6 +135,13 @@ abstract contract MerkleStreamerLL_Fork_Test is Fork_Test {
             "MerkleStreamerLL contract does not match computed address"
         );
 
+        // Check configuration for rebasing asset on Blast L2.
```

Comment:
I have only kept this test here since in the case of Automatic yield, the `clawback` function can be used to claim yield which already has a test below.

All other Sablier-related functions are inherited from v2-core where we have already tested them.

Any thoughts on this?

Reply:
Same here @andreivladbrg right? Same reason as above.